pod/perlfunc.pod

```diff
@@ -7547,14 +7559,26 @@ X<shift>
 
 =for Pod::Functions remove the first element of an array, and return it
 
-Shifts the first value of the array off and returns it, shortening the
-array by 1 and moving everything down.  If there are no elements in the
-array, returns the undefined value.  If ARRAY is omitted, shifts the
-L<C<@_>|perlvar/@_> array within the lexical scope of subroutines and
-formats, and the L<C<@ARGV>|perlvar/@ARGV> array outside a subroutine
-and also within the lexical scopes
-established by the C<eval STRING>, C<BEGIN {}>, C<INIT {}>, C<CHECK {}>,
-C<UNITCHECK {}>, and C<END {}> constructs.
+Removes and returns the B<first> element of an array. This shortens the
+array by one and moves everything down.
+
+    my @arr  = ('cat', 'dog');
+    my $item = shift(@arr); # 'cat'
+
+    # @arr is now ('dog');
+
+Returns C<undef> if the array is empty. If ARRAY is omitted, C<shift>
+operates on the C<@ARGV> array in the main program, and the C<@_> array
+in subroutines.
```

Comment:
I think i  would move the "If ARRAY is ommitted" part down to join the part about eval and the special blocks. 

But what I was really wondering is if you can somehow eliminate the need to duplicate this bit of docs, by maybe making one of the two refer to the other.

Eg, Perhaps "shift" would say 

```
If ARRAY is omitted, C<shift> operates on the the C<@_> array
in normal subroutines, and on the C<@ARGV> array in the main program, and 
in C<eval STRING>, C<BEGIN {}>,  C<INIT {}>, and C<CHECK {}> blocks.
```

And then in the docs for pop it could say

```
If ARRAY is omitted, C<pop> defaults to using either C<@_> or C<@ARGV>, 
exactly as C<shift> does, see L</shift> for details.
```
(module a correct L<> link for shift of course).

Sorry it took me so long to get back to you, I was out of action for a while

Reply:
I don't think it's necessarily worth consolidating with a cross link just to save a line of description.

Reply:
I dont feel very strongly about cross linking or not, but i do feel strongly that the different bits of docs for the "no array specifed" case should be in one sentence, and not part of a sentence referencing what happens if the array is empty.  

The cross linking suggestion was mostly to avoid the two descriptions getting out of sync with each other as I think they were previously, and to avoid double bookkeeping, but I don't feel that strongly about it.

Reply:
I agree with moving things down... I updated the PR.